planning/behavior_velocity_planner/src/scene_module/stop_line/scene.cpp

```diff
@@ -109,6 +109,18 @@ boost::optional<StopLineModule::SegmentIndexWithPoint2d> StopLineModule::findCol
 {
   const size_t min_search_index = std::max(static_cast<size_t>(0), search_index.min_idx);
   const size_t max_search_index = std::min(search_index.max_idx, path.points.size() - 1);
+
+  // for creating debug marker
+  if (planner_param_.show_stopline_collision_check) {
+    debug_data_.search_stopline = stop_line;
+    for (size_t i = min_search_index; i < max_search_index; ++i) {
+      const auto & p_front = path.points.at(i).point.pose.position;
+      const auto & p_back = path.points.at(i + 1).point.pose.position;
```

Comment:
@h-ohta 
can you check if ` p_back = path.points.at(i + 1).point.pose.position;`wont exceeds max index?

Reply:
max_search_index is clipped at https://github.com/autowarefoundation/autoware.universe/pull/932/files#diff-4a562700ada9c3821029f9b18eef63077074de7135ddaadcff94418a3aa15587R111.
It would be enough.